core/src/util/pbfParser.cpp

```diff
@@ -124,7 +130,17 @@ void PbfParser::extractFeature(ParserContext& _ctx, protobuf::message& _featureI
                 break;
         }
     }
-    _out.props = std::move(_ctx.properties);
+
+    std::vector<Properties::Item> properties;
+    properties.reserve(_ctx.featureTags.size());
+
+    for (int tagKey : _ctx.orderedKeys) {
+        int tagValue = _ctx.featureTags[tagKey];
+        if (tagValue >= 0) {
+            properties.emplace_back(_ctx.keys[tagKey], _ctx.values[tagValue]);
+        }
+    }
+    _out.props.setSorted(std::move(properties));
```

Comment:
Oh - no, the Properties constructor that takes vector<Property::Item>&& does sort the vector. I think it may be better to have an explicitly named function for the shortcut